pallets/README.md

```diff
@@ -10,31 +10,69 @@ To run the tests use:
 cargo test --package <pallet-name> --features runtime-benchmarks
 ```
 
+
 ## Benchmarking
 
+### Running manually
+
 Build the node with the `runtime-benchmarks` feature:
 
 ```bash
 cargo build --package spacewalk-standalone --release --features runtime-benchmarks
 ```
 
 ```bash
-# Show all available benchmarks
-./target/release/spacewalk-standalone benchmark pallet --list
-
-# Show benchmarks for a specific pallet
-./target/release/spacewalk-standalone benchmark pallet -p <pallet-name> -e '*' --list
+# Show benchmarks for this pallet
+./target/release/spacewalk-standalone benchmark pallet -p fee -e '*' --list
```

Comment:
Is there a reason you changed it to use the 'fee' pallet in this README? Since this is the 'general' README for all pallets I wouldn't put instructions for a specific pallet in here.